packages/pn-personafisica-webapp/src/pages/SercqSendActivation.page.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { Box, Typography } from '@mui/material';
+import { PnWizard, PnWizardStep } from '@pagopa-pn/pn-commons';
+import { ButtonNaked } from '@pagopa/mui-italia';
+
+import SercqSendContactWizard from '../components/Contacts/SercqSendContactWizard';
+import LoadingPageWrapper from '../components/LoadingPageWrapper/LoadingPageWrapper';
+import { getDigitalAddresses } from '../redux/contact/actions';
+import { resetState } from '../redux/contact/reducers';
+import { useAppDispatch } from '../redux/hooks';
+
+const SercqSendActivation: React.FC = () => {
+  const { t } = useTranslation(['recapiti', 'common']);
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+
+  const [activeStep, setActiveStep] = useState(0);
+  const [pageReady, setPageReady] = useState(false);
+
+  const goToNextStep = () => {
+    setActiveStep(activeStep + 1);
+  };
+
+  const getNextButton = () => {
+    if (activeStep === 0) {
+      return (
+        <ButtonNaked onClick={() => navigate(-1)} color="primary" size="medium" sx={{ mx: 'auto' }}>
+          {t('button.annulla', { ns: 'common' })}
+        </ButtonNaked>
+      );
+    }
+
+    return null;
+  };
+
+  const fetchAddresses = useCallback(() => {
```

Comment:
Maybe we don't need to call the fetch address. The addresses are populated at the application start and when the user lands on recapiti page